index.html

```diff
@@ -1,62 +1,199 @@
-            /></a>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="./style.css" />
+  <title>Personal Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Odibee+Sans&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+</head>
+
+<body>
+
+  <main>
+    <header class="container">
+      <a href="#" class="logo"><img src="./img/Logo_placeholder.png" alt="Logo" /></a>
+      <nav>
+        <a href="#" class="menu"><img src="./img/Enabled.png" alt="Menu button" /></a>
+      </nav>
+    </header>
+    <section class="headline">
+      <h1 class="primary-text">Hey There. I'm Festus</h1>
+      <h2 class="supporting-text">I am a Software Developer</h2>
+      <p class="description">
+        I can help you build a product , feature or website Look through some
+        of my work and experience! If you like what you see and have a project
+        you need coded, don't hestiate to contact me.
+      </p>
+      <ul class="social-media">
+        <li>
+          <a href="#" class="git"><img src="./img/Github.png" alt="Github logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="linkedin"><img src="./img/linkedIn.png" alt="Linkedin logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="angelist"><img src="./img/angelist.png" alt="Angelist logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="twitter"><img src="./img/twitter.png" alt="Twitter logo" /></a>
+        </li>
+        <li>
+          <a href="#" class="vector"><img src="./img/vector.png" alt="Vector logo" /></a>
+        </li>
+      </ul>
+    </section>
+
+      <section class="works-section">
+        <div class="topic-wrapper">
+          <h2 class="topic">My Recent Works</h2>
+          <img src="./img/underline.png" alt="underline" class="underline" />
+        </div>
+        <div class="title-container">
+          <img src="./img/Img_placeholder.png" alt="featured image" class="placeholder" />
+          <div class="title-wrapper">
+            <h3 class="title">Multi-Post Stories</h3>
+            <p class="supporting-text2">A daily selection of privately personalized reads; no accounts or sign-ups
+              required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+              standard dummy text.</p>
+            <ul class="tags">
+              <li class="tag-text">css</li>
+              <li class="tag-text">html</li>
+              <li class="tag-text">bootstrap</li>
+              <li class="tag-text">ruby</li>
+            </ul>
+            <a href="#" target="_blank" class="butt">See Project</a>
```

Comment:
- [ ] [OPTIONAL] - You have done well styling your buttons. They now only lack one small touch to perfection. I recommend that you include button states eg on hover, on click, on active states. Adding these to all your buttons will improve the UI/ UX of your website. Below is how you can implement the design of the state as provided by the chosen [Figma design.](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471). You can have a look at [this resource](https://www.w3schools.com/css/css_link.asp) if you are not sure how this can be achieved.

**Figma button states:**
![image](https://user-images.githubusercontent.com/86318284/176993376-0682dc61-7047-40fa-addb-bae1544c7d40.png)

**Kindly do this on all buttons or links**